external.bzl

```diff
@@ -16,15 +16,16 @@ def go_dependencies():
     go_repository(
         name = "com_github_alicebob_gopher_json",
         importpath = "github.com/alicebob/gopher-json",
-        sum = "h1:45bxf7AZMwWcqkLzDAQugVEwedisr5nRJ1r+7LYnv0U=",
-        version = "v0.0.0-20180125190556-5a6b3ba71ee6",
+        sum = "h1:HbKu58rmZpUGpz5+4FfNmIU+FmZg2P3Xaj2v2bfNWmk=",
+        version = "v0.0.0-20200520072559-a9ecdc9d1d3a",
     )
     go_repository(
-        name = "com_github_alicebob_miniredis",
-        importpath = "github.com/alicebob/miniredis",
-        sum = "h1:yBHoLpsyjupjz3NL3MhKMVkR41j82Yjf3KFv7ApYzUI=",
-        version = "v2.5.0+incompatible",
+        name = "com_github_alicebob_miniredis_v2",
+        importpath = "github.com/alicebob/miniredis/v2",
+        sum = "h1:QWoo2wchYmLgOB6ctlTt2dewQ1Vu6phl+iQbwT8SYGo=",
+        version = "v2.14.3",
     )
+
```

Comment:
nit, and this really is a bug in gazelle that it's inconsistent I should report: when it first generate this file it doesn't add those blank lines, and then when it makes changes it adds those blank lines after every block it changed. currently I usually just manually remove those blank lines.

Reply:
what I meant is that it would be great if you can manually remove those added blank lines 😅

Reply:
![image](https://user-images.githubusercontent.com/1929108/109087486-6975e880-76c2-11eb-9c8f-ed4137755048.png)

My brain didn't look closely enough at this and I thought you were talking about a newline at the end of the file...